v4/source/teams.yaml

```diff
@@ -1145,6 +1145,7 @@
 
             $resp = $driver->getTeamModel()->addUser($teamID, [
```

Comment:
Is it needed to pass the team id twice? It is being passed here already.

Reply:
I agree, this does not seem very useful. But the PHP API works like this for the moment, see the details in the linked issue (https://github.com/mattermost/mattermost-api-reference/issues/552)

Reply:
Thanks! Sorry for not checking the issue previously :sweat_smile: